collections/so2_daily_data.yaml

```diff
@@ -0,0 +1,81 @@
+Name: SO2 daily data
+Description: |
+  This online platform uses data from the Copernicus Sentinel-5P satellite and shows the daily sulfur dioxide concentrations coming primarily from volcanic sources. The Copernicus Sentinel-5P SO2 measurements are those retrieved assuming SO2 at an altitude of 7km and explicitly filtering for pixels where a volcanic source is most likely (sulfurdioxide_detection_flag > 0) and where the solar zenith angle is within limits (SZA < 70°). The measurements are then mapped on a fixed latitude-longitude grid of 8193 x 16385 pixels. The grid is turned into an EPSG:4326 geotiff file using the appropriate color scale, which is again turned into an EPSG:3857 tile map.
+  Note that for data shown, SO2 from potential anthropogenic sources has not been filtered out, and SO2 from such sources can therefore still show up.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: so2_daily_data/README.MD
+Image: so2_daily_data/so2.png
+EDCBrowser: ""
+Resolution: 4.8 x 4.8 Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 2021-09-19 - ongoing
+UpdateFrequency: daily
+BandInformation: so2
```

Comment:
Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure).